### PR TITLE
Kindle.Create version 1.63.0.0

### PR DESCRIPTION
**Pull request by weeyin83**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45090)

### CODE REVIEW

File: manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Kindle.Create
```
Comment (OfficialEsco) [inline manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.installer.yaml:4]:
```suggestion
PackageIdentifier: Amazon.KindleCreate
```

File: manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.locale.en-us.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Kindle.Create
```
Comment (OfficialEsco) [inline manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.locale.en-us.yaml:4]:
```suggestion
PackageIdentifier: Amazon.KindleCreate
```

File: manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Kindle.Create
```
Comment (OfficialEsco) [inline manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.yaml:4]:
```suggestion
PackageIdentifier: Amazon.KindleCreate
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Whats the difference between Kindle Create and Amazon Kindle?
Since Amazon is the Publisher is should go under the same Publisher folder, and Software folder `KindleCreate` or `Kindle/Create` depending how related these are

**Comment by OfficialEsco** [comment]
You missed one, but you also have to move the folders

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/45090#issuecomment-1031894014

File: manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.locale.en-us.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Kindle.Create
+PackageVersion: 1.63.0.0
+PackageLocale: en-us
+Publisher: Amazon
+PackageName: Kindle Create
+License: MIT License
```
Comment (KaranKad) [inline manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.locale.en-us.yaml:9]:
```suggestion
License: Proprietary
```
Is this really MIT License? Proprietary seems the right one

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by weeyin83** [comment]
Close

**Pull request closed without merging**